nixos/doc/manual/release-notes/rl-2009.xml

```diff
@@ -235,7 +235,16 @@ php.override {
        Be aware that backwards state migrations are not supported by Deluge.
      </para>
    </listitem>
-
+   <listitem>
+     <para>
+       Add option <literal>services.nginx.enableSandbox</literal> to starting Nginx web server with additional sandbox/hardening options.
+       By default, write access to <literal>services.nginx.stateDir</literal> is allowed. To allow writing to other folders,
```

Comment:
I think that the release notes should also have mentioned that due to the added `ProtectHome = mkDefault true;`, nginx can no longer serve files from users' home directories.

It's not uncommon to serve subdirectories of home directories publicly (e.g. `/home/*/public_html/`), and this change breaks that without any mention.